### PR TITLE
Add CapMetro provider

### PR DESCRIPTION
**Pull request by colinjmurphy**
Implements the abstract HAFAS provider for CapMetro, the transit authority for Austin, Texas.

### CODE REVIEW

File: enabler/src/de/schildbach/pte/CapMetroProvider.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import okhttp3.HttpUrl;
+
+/**
+ * @author Colin Murphy
+ */
+public class CapMetroProvider extends AbstractHafasMobileProvider {
+    private static final HttpUrl API_BASE = HttpUrl.parse("https://capmetro.hafas.cloud/bin/");
+
+    // MetroRail: 8, MetroBus: 32, MetroRapid: 4096
+    private static final Product[] PRODUCTS_MAP = {
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/CapMetroProvider.java]: Are there really no other products?

File: enabler/test/de/schildbach/pte/live/CapMetroLiveTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package de.schildbach.pte.live;
```
Comment (schildbach) [inline enabler/test/de/schildbach/pte/live/CapMetroLiveTest.java]: Can you add the license here too?

File: enabler/src/de/schildbach/pte/CapMetroProvider.java (hunk 1 above)
Comment (derhuerst) [reply enabler/src/de/schildbach/pte/CapMetroProvider.java]: Fidn't find any either via reverse-engineering: https://github.com/public-transport/hafas-client/blob/c6b7aa74ddef57c5531623b95f6f70333368f34f/p/cmta/products.js

Comment (schildbach) [reply enabler/src/de/schildbach/pte/CapMetroProvider.java]: Ok, thanks.

File: enabler/test/de/schildbach/pte/live/CapMetroLiveTest.java (hunk 2 above)
Comment (colinjmurphy) [reply enabler/test/de/schildbach/pte/live/CapMetroLiveTest.java]: Definitely.

**Comment by colinjmurphy** [comment]
Made the suggested changes, thanks for taking a look.

**Comment by schildbach** [comment]
I renamed the classes to `Cmta*` as well, added a timezone and merged. Thanks a lot!

**Pull request closed without merging**